datadog-slos-slo-handler/datadog-slos-slo.json

```diff
@@ -0,0 +1,207 @@
+{
+  "typeName": "Datadog::SLOs::SLO",
+  "description": "Datadog SLO 1.0.0",
+  "definitions": {
+    "Creator": {
+      "type": "object",
+      "properties": {
+        "Name": {
+          "description": "Name of the creator of the slo",
+          "type": "string"
+        },
+        "Handle": {
+          "description": "Handle of the creator of the slo",
+          "type": "string"
+        },
+        "Email": {
+          "description": "Email of the creator of the slo",
+          "type": "string"
+        }
+      }
+    },
+    "Threshold": {
+      "type": "object",
+      "properties": {
+        "Target": {
+          "description": "TThe target value for the service level indicator within the corresponding timeframe.",
+          "type": "number"
+        },
+        "TargetDisplay": {
+          "description": "TA string representation of the target that indicates its precision.(e.g. 98.00)",
+          "type": "string"
+        },
+        "Timeframe": {
+          "description": "The SLO time window options. Allowed enum values: 7d,30d,90d",
+          "type": "string",
+          "enum": [
+            "7d",
+            "30d",
+            "90d"
+          ]
+        },
+        "Warning": {
+          "description": "The warning value for the service level objective.",
+          "type": "number"
+        },
+        "WarningDisplay": {
+          "description": "A string representation of the warning target.(e.g. 98.00)",
+          "type": "string"
+        }
+      }
+    },
+    "Query": {
+      "type": "object",
+      "properties": {
+        "Numerator": {
+          "description": "A Datadog metric query for total (valid) events.",
+          "type": "string"
+        },
+        "Denominator": {
+          "description": "A Datadog metric query for good events.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "properties": {
+    "DatadogCredentials": {
+      "description": "Credentials for the Datadog API",
+      "properties": {
+        "ApiKey": {
+          "description": "Datadog API key",
+          "type": "string"
+        },
+        "ApplicationKey": {
+          "description": "Datadog application key",
+          "type": "string"
+        },
+        "ApiURL": {
+          "description": "Datadog API URL (defaults to https://api.datadoghq.com) Use https://api.datadoghq.eu for EU accounts.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "ApiKey",
+        "ApplicationKey"
+      ],
+      "type": "object"
+    },
```

Comment:
Now that Amazon has release their new TypeConfiguration feature for specifying 3rd party credentials, can you update this ?
You can look at the following for an example of how we've done it https://github.com/DataDog/datadog-cloudformation-resources/pull/142/files#diff-c31db319bea7f57ab58a39a286158589a540cc82eacc2c817055b1381a1eb0b4R4-R10

Reply:
This has been updated